### PR TITLE
Loading Bar Animation

### PR DESCRIPTION
**Pull request by okorohelijah**
This loads the posts and when shows the feeds when the feed is done loading

https://user-images.githubusercontent.com/89285052/179628399-45a7d384-5e0f-4304-bfc1-f7631aa2b632.mp4

### CODE REVIEW

File: app/src/main/java/com/example/yard/activities/YardAnimationActivity.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package com.example.yard.activities;
+
+import android.animation.Animator;
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.appcompat.app.AppCompatActivity;
+
```
Comment (CatWang) [inline app/src/main/java/com/example/yard/activities/YardAnimationActivity.java:8]: remove the new lines during import

File: app/src/main/java/com/example/yard/activities/YardAnimationActivity.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      i.setClass(getApplicationContext(), LoginActivity.class);
+      LottieAnimationView lottieView = findViewById(R.id.YardAnimation);
+      lottieView.addAnimatorListener(
+          new Animator.AnimatorListener() {
+            @Override
+            public void onAnimationStart(Animator animator) {}
+
+            @Override
+            public void onAnimationEnd(Animator animator) {
+              // do here the action you want
```
Comment (CatWang) [inline app/src/main/java/com/example/yard/activities/YardAnimationActivity.java:31]: remove the unnecessary comment

**Review by CatWang — APPROVED** [review]
Nicely organized PR! Great job!

Can we add a demo video to the PR description?

File: app/src/main/java/com/example/yard/activities/YardAnimationActivity.java (hunk 1 above)
Comment (okorohelijah) [reply app/src/main/java/com/example/yard/activities/YardAnimationActivity.java:8]: https://source.android.com/devices/architecture/hidl/code-style - Android Studio Code Style Documentation groups the import statements by its group like all the 'android.os' in one group then a space then the ones with 'androidx' in another group

**Pull request merged**